pkg/tsdb/opentsdb/opentsdb.go

```diff
@@ -53,7 +57,7 @@ func (s *Service) Init() error {
 		QueryDataHandler: s,
 	})
 
-	if err := s.BackendPluginManager.Register("opentsdb", factory); err != nil {
+	if err := s.BackendPluginManager.RegisterAndStart(context.Background(), "opentsdb", factory); err != nil {
```

Comment:
It's sadly unclear from the API but this should be addressed soon - but Register is most appropriate here 😀

Reply:
ok, at this point influxdb, graphite, loki, tempo and cloudwatch has RegisterAndStart, is that a problem?

Reply:
No it's okay - under the hood they do the same thing. We can have a separate PR for that